Specs/DateTime/English/DateParser.json

```diff
@@ -2355,4 +2355,76 @@
     }
   ]
 }
+,
+{
+  "Input": "I'll see you the first Friday of next month",
```

Comment:
Can you add tests for “first Thursday of next year”?
Also, there are already tests for “Tuesday next week”, “Sunday of next week”, and similar, right?

Reply:
“first Thursday of next year” is not recognized, I'll fix it.
“Tuesday next week”, “Sunday of next week” are supported, I'll add some tests for them.